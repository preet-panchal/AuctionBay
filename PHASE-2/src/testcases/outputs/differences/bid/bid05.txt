--- ./../testcases/outputs/expected/bid/bid05.txt	2023-03-06 15:16:23
+++ ./../testcases/outputs/actual/bid/bid05.txt	2023-03-06 15:53:07
@@ -13,14 +13,15 @@
 
 Enter operation:
 Enter username:
-Hello admin01 (AA)!
+Hello User01 (AA)!
 
 Enter operation:
 Enter item name:
 Enter seller's username:
-Current Highest Bid: $100.00
-Enter bid amount:
-Bid of $200.00 placed on item01.
+Error: item01 does not exist in available-items file.
 
 Enter operation:
-admin01 successfully logged out.
+Invalid input for operation. Please try again.
+
+Enter operation:
+User01 successfully logged out.
