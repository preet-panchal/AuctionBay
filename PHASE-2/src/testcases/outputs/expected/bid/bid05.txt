Welcome to AuctionBay!
--
Here is a list of operations (what to input):
- Login (login)
- Create (create)
- Delete (delete)
- Advertise (advertise)
- Bid (bid)
- Refund (refund)
- Add Credit (addcredit)
- List All Items (listall)
- Logout (logout)

Enter operation:
Enter username:
Hello admin01 (AA)!

Enter operation:
Enter item name:
Enter seller's username:
Current Highest Bid: $100.00
Enter bid amount:
Bid of $200.00 placed on item01.

Enter operation:
admin01 successfully logged out.
